client/src/components/participant-details/psi-cohort.js

```diff
@@ -97,39 +93,43 @@ const PSIRouteTabs = ({
         value={tab}
         onChange={(_, prop) => {
           setTab(prop);
-          history.push(TabDetails[prop]?.path);
+          history.push(tabDetails[prop]?.path);
         }}
       >
-        {Object.keys(TabDetails).map((key) => (
+        {Object.keys(tabDetails).map((key) => (
           <CustomTab
             key={key}
-            label={TabDetails[key].label}
+            label={tabDetails[key].label}
             value={key}
             disabled={isLoadingData}
           ></CustomTab>
         ))}
       </CustomTabs>
       <Switch>
-        <Route exact path={TabDetails.trackGraduation.path}>
-          <TabContentTrackGraduation
-            tab={tab}
-            setTab={setTab}
-            participant={participant}
-            fetchData={fetchData}
-          />
-        </Route>
-
-        <Route exact path={TabDetails.assignCohort.path}>
-          <TabContentAssignCohort
-            tab={tab}
-            setTab={setTab}
-            fetchData={fetchData}
-            assignAction={assignAction}
-            disabled={disabled}
-            psiList={psiList}
-          />
-        </Route>
-        <Redirect to='/' />
+        {tabDetails.trackGraduation &&
```

Comment:
I don't love this either, I would've rather included the component in `tabDetails` or something, but I'm open to suggestions